examples/src/app/code-editor.tsx

```diff
@@ -37,7 +37,11 @@ const CodeEditor = (props: CodeEditorProps) => {
 
     const beforeMount = (monaco: any) => {
         monaco.languages.typescript.typescriptDefaults.addExtraLib(
-            playcanvasTypeDefs,
+            playcanvasTypeDefs + ' export as namespace pc;',
```

Comment:
Doesn't this suggest there's something missing from the `playcanvas.d.ts` file? `export as namespace pc` used to be in the old types but I thought maybe it was unneeded. I can add it back if necessary?

Reply:
Yeah that'd be good as this used to be present.

Reply:
I've updated the rollup config as suggested.